scripts/create.sh

```diff
@@ -38,7 +38,7 @@ run_terraform() {
 
 generate_static_ip() {
     # Check to make sure static IP hasn't already been created, then register it.
-    if [[ $(gcloud compute addresses list | grep 'prime-server') = '' ]]; then
+    if [[ $(gcloud compute addresses list | grep ' prime-server') = '' ]]; then
```

Comment:
It might be better to use Regex for this task?

```bash
^NAME:\sprime-server
```

Reply:
Sounds good. Changed the pattern.